Makefile

```diff
@@ -4,6 +4,8 @@ VERSION ?= 0.1.3
 BUNDLE_IMG ?= controller-bundle:$(VERSION)
 #operator-sdk version
 OPERATOR_SDK_VERSION ?= v1.4.2
+#Options for Template-Validator
+VALIDATOR_VERSION ?= V0.9.0
```

Comment:
The validator version could be the same as ssp-operator version. `${VERSION}`.
The version should also be used in `internal/operands/template-validator/defaults.go`, so that the SSP operator deploys the same validator version.

Reply:
Changed now